packages/flutter_tools/test/commands.shard/hermetic/build_macos_test.dart

```diff
@@ -191,6 +191,33 @@ void main() {
     Platform: () => macosPlatform,
     FeatureFlags: () => TestFeatureFlags(isMacOSEnabled: true),
   });
+  testUsingContext('macOS build supports build name and number', () async {
```

Comment:
nit: newline between previous test case
```suggestion

  testUsingContext('macOS build supports build name and number', () async {
```

Reply:
👍️